README.md

```diff
@@ -6,9 +6,37 @@ An extension for Google's [AutoValue](https://github.com/google/auto) that creat
 
 ## Usage
 
-Simply include auto-value-moshi in your project and add a public static method with the following
-signature to classes you want to get Moshi `JsonAdapter`s. You can also annotate your properties
-using `@Json` to define an alternate name for de/serialization.
+Simply include auto-value-moshi in your project and annotate your target autovalue class with Moshi's
+`@JsonClass` annotation. `generateAdpater` must be true, and the `generator` property value should
+be `"avg"`.
```

Comment:
Should be "autovalue", or at least "avm", right?

Reply:
d'oh, fixed avm